main.go

```diff
@@ -41,9 +41,9 @@ import (
 	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
 	loggingreconciler "github.com/giantswarm/logging-operator/pkg/logging-reconciler"
 	"github.com/giantswarm/logging-operator/pkg/reconciler"
-	grafanaagentconfig "github.com/giantswarm/logging-operator/pkg/resource/grafana-agent-config"
```

Comment:
to split this into smaller chunks. You could rename the packages to event-logging-config and event-logging-secret first?

Reply:
But that's already the case right ? (except for the naming `k8s-events` that I need to update to `events-logger`

Reply:
You do a migration PR first

Reply:
Smaller PR : https://github.com/giantswarm/logging-operator/pull/269